src/fiskaltrust.Launcher.Common/Configuration/Configuration.cs

```diff
@@ -27,29 +27,30 @@ public record LauncherConfiguration
     {
         public const string DATA_PROTECTION_DATA_PURPOSE = "fiskaltrust.Launcher.Configuration";
 
-        private bool _useDefaults;
+        private bool _raw;
 
         [JsonConstructor]
-        public LauncherConfiguration() { _useDefaults = false; }
+        public LauncherConfiguration() { _raw = false; }
 
-        public LauncherConfiguration(bool useDefaults)
+        public T Raw<T>(System.Linq.Expressions.Expression<Func<LauncherConfiguration, T>> accessor)
         {
-            _useDefaults = useDefaults;
-        }
-
-        public void EnableDefaults()
-        {
-            _useDefaults = true;
-        }
-
-        public void DisableDefaults()
-        {
-            _useDefaults = false;
+            lock (this)
```

Comment:
Not super critical in this scope, but generally I think it's not a super good practice to lock on `this` (because someone outside of this class could theoretically also lock on that object) - but we can definitely keep that here 😅

Reply:
So, best practice would be a private lock field